src/pages/Profile/profile.css

```diff
@@ -0,0 +1,26 @@
+.feed-container {
+  width: 60vw;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+}
+.edit-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2rem;
+  padding: 2rem;
+}
+.form-element {
+  display: flex;
+  flex-direction: column;
```

Comment:
I can see this style is being used multiple times hence making your code redundant. You can create a small utility for flex that will help you at many places as you're heavily using it

Reply:
I agree, will refactor this in coming versions.